invenio_app_ils/ui/backoffice/src/common/api.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+
+// In development create a personal token to be able to perform requests and
+// and put it in a .env file under the name `REACT_APP_JWT_TOKEN`
+const token =
```

Comment:
this is gonna fail on initialization time on production  I think. For production I would export a function and set it manually from the request. So something like 

```

// In development create a personal token to be able to perform requests and
// and put it in a .env file under the name `REACT_APP_JWT_TOKEN`

const environment = process.env.NODE_ENV === 'production' ? 'production' : 'development';
const devToken = environment === 'development' ? process.env.REACT_APP_JWT_TOKEN : '';

const axiosIls = axios.create({
  baseURL:
    process.env.NODE_ENV === 'development'
      ? 'https://localhost:5000/api'
      : '/api',
  headers: {
    Authorization: `bearer ${token}`,
  },
  withCredentials: true,
});

setBearer() {
  let bearer = environment == 'production' ? (document.getElementsByName('authorized_token') || '') : devToken;
  if(bearer.length) {
    bearer = bearer[0].value;
    return {
      headers: {
        Authorization: `bearer ${bearer}`
      }
    }
  }
}
export const fetchRecord = (url, recid) => {
  return axiosIls.get(`${url}/${recid}`, setBearer());
};

export const postRecord = (url, data) => {
  return axiosIls.post(url, data, setBearer());
}
```